nexus/db-queries/src/db/datastore/ip_pool.rs

```diff
@@ -81,6 +81,7 @@ impl DataStore {
         )
         .filter(ip_pool::time_deleted.is_null())
         .select(IpPool::as_select())
+        .distinct()
```

Comment:
This is the easy fix, but I'm wondering if instead of using linkage to the internal silo to identify the internal pool, we could leave out the join altogether if we just had a well-known name for the internal pool and filtered that out.

Reply:
I believe the `SERVICE_IP_POOL_NAME` const should be usable for this.

https://github.com/oxidecomputer/omicron/blob/009c4ae24d7394550f779dc02d83097293d87873/nexus/db-queries/src/db/datastore/mod.rs#L117

https://github.com/oxidecomputer/omicron/blob/009c4ae24d7394550f779dc02d83097293d87873/nexus/db-queries/src/db/datastore/rack.rs#L756-L760

Reply:
Will attempt.